Error at %TEST_DIR%/typechecker/try/error_return_type_err_mismatch.abra:4:11
Return type mismatch for function containing try expression
  |    val x = try foo()
               ^
The containing function 'bar' returns a Result whose error type is 'Int', which is incompatible with the try expression.
  |  func bar(): Result<Int, Int> {
          ^
To be compatible, the return type of 'bar' must have 'String' as its error type.
Hint: You can use an else-clause to transform the error value into a compatible value and return it
